src/CMakeLists.txt

```
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.15)
 
 project(sudoManager)
-add_library(${PROJECT_NAME} SHARED Menu.cpp Item.cpp)
+add_library(${PROJECT_NAME} SHARED Menu.cpp Item.cpp Bill.cpp Payment.cpp)
 
 target_include_directories(${PROJECT_NAME} PUBLIC ${CMAKE_SOURCE_DIR}/include)
 target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_20)
```

```suggestion
target_compile_features(${PROJECT_NAME} PUBLIC cxx_std_20)
```
This will fix your `Ubuntu-20-04-x86_64` check fail